hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/map/MapMadePublishableMessageTask.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.client.impl.protocol.task.map;
+
+import com.hazelcast.client.impl.protocol.ClientMessage;
+import com.hazelcast.client.impl.protocol.codec.EnterpriseMapMadePublishableCodec;
+import com.hazelcast.client.impl.protocol.task.AbstractAllPartitionsMessageTask;
+import com.hazelcast.instance.Node;
+import com.hazelcast.map.impl.MapService;
+import com.hazelcast.map.impl.querycache.subscriber.operation.MadePublishableOperationFactory;
+import com.hazelcast.nio.Connection;
+import com.hazelcast.spi.OperationFactory;
+
+import java.security.Permission;
+import java.util.Map;
+
+/**
+ * Client Protocol Task for handling messages with type id:
+ * {@link com.hazelcast.client.impl.protocol.codec.EnterpriseMapMessageType#ENTERPRISEMAP_MADEPUBLISHABLE}
+ */
+public class MapMadePublishableMessageTask
+        extends AbstractAllPartitionsMessageTask<EnterpriseMapMadePublishableCodec.RequestParameters> {
+
+    public MapMadePublishableMessageTask(ClientMessage clientMessage, Node node, Connection connection) {
+        super(clientMessage, node, connection);
+    }
+
+    @Override
+    protected EnterpriseMapMadePublishableCodec.RequestParameters decodeClientMessage(ClientMessage clientMessage) {
```

Comment:
Can we rename it like this: `EnterpriseMapMadePublishableCodec`-->`MapMadePublishableCodec `?